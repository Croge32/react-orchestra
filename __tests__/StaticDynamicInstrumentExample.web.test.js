
import React from 'react';
import renderer from 'react-test-renderer';
// jest.dontMock('../demo/src/components/static-instrument-example');


import StaticInstrument from '../demo/src/components/static-instrument-example';
import InteractiveInstrument from '../demo/src/components/interactive-instrument-example';
import { shallow, mount, render } from 'enzyme';

const delay = ms => new Promise(resolve => setTimeout(resolve, ms));

test('StaticInstrument demo renders correctly', async () => {
  // console.log(a);

  const component = renderer.create(
    <div>
      <StaticInstrument />
      <InteractiveInstrument />
    </div>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
