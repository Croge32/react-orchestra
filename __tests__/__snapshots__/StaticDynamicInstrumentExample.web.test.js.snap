exports[`test StaticInstrument demo renders correctly 1`] = `
<div>
  <div>
    <div
      className="ro-instrument-loading"
      style={undefined}>
      <div>
        <span>
          Loading  Instrument ðŸšš ðŸšš ðŸšš
        </span>
      </div>
      <div
        className=""
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}>
        <div>
           This is what I want my note to look like ! I can put anything in here.
          <img
            alt="Some pic"
            src="https://s-media-cache-ak0.pinimg.com/originals/36/43/e7/3643e7e8dab9b88b3972ee1c9f909dea.jpg"
            width="80" />
        </div>
      </div>
      <div
        className=""
        onMouseDown={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}>
        <div>
          Another note
        </div>
      </div>
    </div>
    <div
      onClick={[Function]}>
      Play melody
    </div>
  </div>
  <div
    className="ro-instrument-loading"
    style={undefined}>
    <div>
      <span>
        Loading  Instrument ðŸšš ðŸšš ðŸšš
      </span>
    </div>
    <div
      className=""
      onMouseDown={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}>
      <div>
        <div>
          Click me to play A3
        </div>
      </div>
    </div>
    <div
      className=""
      onMouseDown={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}>
      <div>
        Click me to play C3
      </div>
    </div>
  </div>
</div>
`;
